module/userProjects/helper.js

```diff
@@ -3273,6 +3273,75 @@ module.exports = class UserProjectsHelper {
         })
     }
 
+    /**
+     * Fetches project details based on the provided query.
+     * @method
+     * @name details
+     * @param {Object} args - Query object for fetching project details.
+     * @returns {Array} - A promise that resolves to an array of project details matching the query.
+     */
+    static userProjectOverview(args,fields='all',stats) {
+          return new Promise(async (resolve, reject) => {
+            try{
+    
+                const projectIdQuery = {
+                    ...args
+                  };
+
+                  let fieldarray = []
+           
+                  if (fields != "all") {
+                        fields.forEach(field => {
+                        fieldarray.push(field)
+                     });
+                 }
+
+                if(stats == 'true')
+                {
+                    let count = await projectQueries.countDocuments(projectIdQuery);
+
+                    resolve(count);
+                }
+
+                let projects = await projectQueries.projectDocument(projectIdQuery,fieldarray);
+                
+                resolve(projects)
+            }catch(error){
+                reject(error)
+            }
+          });
+    }
+    static listUserProjects({userId,userInvolvement,stats}){
+
+        return new Promise(async (resolve, reject) => {
+            try{
+                switch(userInvolvement)
+                {
+                    case 'creator':
```

Comment:
the projects collection will have projects consumed by user there is no difference that project creator or consumed both are same only @borkarsaish65

Reply:
removed the logic